guides/integrate-nextauth.md

```diff
@@ -0,0 +1,427 @@
+
+<Tabs groupId="nextjs" queryString>
+<TabItem value="nextauth" label="NextAuth.js">
+
+```javascript
+import { useEffect, useState } from "react";
+import "bootstrap/dist/css/bootstrap.css";
+import { signIn } from "next-auth/react";
+import { Passkey } from "@beyondidentity/bi-sdk-js";
+
+const BIBindPasskey = () => {
+  const [bindPasskeyResult, setBindPasskeyResult] = useState('');
+
+  useEffect(() => {
+    // -- 1
+    const bindPasskey = async () => {
+      const BeyondIdentityEmbeddedSdk = await import("@beyondidentity/bi-sdk-js");
+      let embedded = await BeyondIdentitySdk.EmbeddedSdk.initialize();
+      if (embedded.isBindPasskeyUrl(window.location.href)) {
+        // Only bind passkey if the URL is a "bind" URL
+        let bindPasskeyUrl = window.location.href;
+        // -- 2
+        embedded
+          .bindPasskey(bindPasskeyUrl)
+          .then((result) => {
+            // -- 3
+            setBindPasskeyResult(result);
+            signIn('beyondidentity', {
+              tenant_id: tenantId,
+            });
+          })
+          .catch((error) => {
+            setBindPasskeyResult(error.toString());
+          });
+      }
+    };
+
+    bindPasskey().catch(console.error);
+  }, []);
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        height: '100vh',
+      }}
+    >
+      <div className="container">
+         <div className="row">	
+          <div className="d-flex justify-content-center">	
+            <div className="spinner-border" role="status">	
+              <span className="sr-only"></span>	
+            </div>	
+          </div>	
+        </div>
+        <div className="row">
+          {bindPasskeyResult.length > 0 && (
+            <div className="row row-cols-1 row-cols-md-1 mt-3">
+              <div className="col">
+                <code>
+                  {JSON.stringify(bindPasskeyResult, null, 2)}
+                </code>
+              </div>
+            </div>
+          )}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default BIBindPasskey;
+```
+
+</TabItem>
+<TabItem value="auth" label="Auth.js">
+
+:::warning WIP
+@auth/nextjs is work in progress. For now, please use [NextAuth.js](?nextjs=nextauth#bind-passkey-to-device).
+:::
+
+</TabItem>
+</Tabs>
+
+What's happening here?
+
+1. The `useEffect` is only called once on page load. In this function, we initialize the Beyond Identity SDK and use `embedded.isBindPasskeyUrl` to check if the current page that was redirected to is in fact a valid `bind` URL.
+2. If the URL is valid, we pull the URL using `window.location.href` and pass that directly into `embedded.bindPasskey` to complete the binding process.
+3. Finally, the response of `embedded.bindPasskey` contains a `passkey` object, which represents the passkey bound to the device.
```

Comment:
This part might make sense to keep in the NextAuth tab to it's not shown when Auth.js is selected

Reply:
Makes sense to me. Done.